Client/src/main/java/org/projectcardboard/client/view/battleview/CardAnimation.java

```diff
@@ -48,29 +51,45 @@ public CardAnimation(Group group, ICard card, double y, double x) {
         Playlist playlist;
         Image image;
         if (CardType.SPELL.equals(card.getType())) {
-            image = cachedImages.computeIfAbsent(card.getSpriteName(), key -> ImageLoader.load("/icons/" + card.getSpriteName() + ".png"));
+            String path = card.getIsCustom() ? Config.getInstance().getPathToCustomCardSprites() : "/icons/";
```

Comment:
have you heard of "Dry"  ?  (don't repeat yourself)

At a quick glance the only difference I can see between the two cases is the path.   In the abstract, the current structure looks like this (pseudocode): 

```
if x:
       pathX
       logic
if y:
       pathY
       logic
```

But could be refactored to:

```
path = pathX  if x  else pathY
logic
```